Bio/AlignIO/MauveIO.py

```diff
@@ -46,27 +46,32 @@
 would probably load this using the Bio.AlignIO.parse() function:
 
     >>> from Bio import AlignIO
-    >>> align = AlignIO.parse("Mauve/simple.xmfa", "mauve")
+    >>> align = AlignIO.parse("Mauve/simple_short.xmfa", "mauve")
     >>> alignments = list(align)
     >>> for aln in alignments:
-    ...     print(align)
+    ...     print(aln)
```

Comment:
Would ideally include another line here ``...`` as you would see in the Python terminal to close the for loop.

Reply:
Yes, I did it in other places, but when the tests passed I forgot about it. Will do.